Controllers/HomeController.cs

```diff
@@ -1,16 +1,32 @@
-﻿using System;
+﻿using OrderManagement_DataLayer;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Web;
 using System.Web.Mvc;
+using OrderManagement.Mapper;
+using OrderManagement.Models;
+using System.Threading;
+using Order = OrderManagement.Models.Order;
```

Comment:
Remove this lines "using Order = OrderManagement.Models.Order;" and  "using OrderManagement_DataLayer;"